service/instance.py

```diff
@@ -947,10 +943,9 @@ def boot_volume_instance(
     """
     kwargs, userdata, network = _pre_launch_instance(
         driver, identity, size, name, **kwargs)
-    kwargs.update(prep_kwargs)
```

Comment:
badly cut and pasted code, probably originally from lines 980-986.  More extensive comment below.